templates/rhpam75-kieserver-externaldb.yaml

```diff
@@ -32,7 +32,7 @@ parameters:
   value: myapp
   required: true
 - displayName: Maven mirror URL
-  description: Maven mirror that the KIE server must use. If you configure a mirror, this mirror must contain all artifacts that are required for building and deploying your services.
+  description: Maven mirror that the KIE server must use. If you configure a mirror, this mirror must contain all artifacts that are required for deploying and running your services.
```

Comment:
this changes sounds not ok for me.

Reply:
How? This template does not build anything.

Reply:
"for deploying your...." sounds enough to me,.